tide_event.install

```diff
@@ -294,3 +294,30 @@ function tide_event_update_8004() {
   _tide_import_single_config('core.entity_view_display.node.event.default', $config_location);
   _tide_import_single_config('field.field.node.event.field_show_related_content', $config_location);
 }
+
+/**
+ * Grant Webform Configuration access to API requests [SDPA-1213].
+ */
+function tide_event_update_8005() {
+  $webforms = \Drupal::entityTypeManager()
+    ->getStorage('webform')
+    ->loadMultiple();
```

Comment:
@anthony-malkoun tide_event should only update its own webform (eg. tide_event_submission), not all existing webforms.

Reply:
OK, @sonnykt. Done!